docs/verifier/query-builder.md

```diff
@@ -2,59 +2,209 @@
 id: query-builder
 title: Query Builder
 sidebar_label: Query Builder
-description: Learn how to use the Query Builder..
+description: Learn how to use the Query Builder.
 keywords:
   - docs
   - polygon id
   - ID holder
   - issuer
   - verifier
   - query builder
+  - credentialAtomicQuerySigV2
+  - credentialAtomicQuerySigV2OnChain
+  - credentialAtomicQueryMTPV2
+  - credentialAtomicQueryMTPV2OnChain
+  - credentialAtomicQueryV3
+  - credentialAtomicQueryV3OnChain
+  - selective disclosure
+  - proof of uniqueness
+  - nullifier
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The [Query Builder](https://schema-builder.polygonid.me/query-builder) is an interactive way of creating verification queries through an intuitive form with all the fields a comprehensive query might have.
+The [Query Builder](https://schema-builder.polygonid.me/query-builder) is a dynamic, user-friendly tool designed for crafting verification queries. It simplifies the process by providing a comprehensive form, encompassing all necessary fields for creating detailed queries.
 
-Verifiers can create a set of rules that users need to meet to prove their identity using their existing credentials. These rules could range from being a member of a specific organization to being at a certain age. Polygon ID makes it easy and secure for users to prove their identity by using these rules for authentication purposes.
+This tool enables verifiers to establish specific rules for identity verification, catering to a range of criteria from organizational membership to age requirements. Polygon ID enhances this process, offering a secure and streamlined way for users to authenticate their identity based on these criteria.
 
-By providing a standardized interface for query creation, a user interface can help ensure that queries are created in a consistent, uniform and error-free way across different users and organizations. Also, rather than requiring users to learn the ZK query language, a user interface can serve as a more intuitive and user-friendly alternative to create queries.
+With its standardized interface, the Query Builder ensures uniformity, consistency, and minimizes errors in query creation across various users and organizations. It also presents a more intuitive alternative to the [ZK Query Language](./verification-library/zk-query-language.md), making query creation accessible even for those without in-depth technical knowledge.
```

Comment:
> It also presents a more intuitive alternative to the ZK Query Language.

It's not an alternative to the ZK Query Language, it's a tool to create queries using the ZK Query Language without having to learn its specifics.

Reply:
What about:

> It also presents a more intuitive alternative to manually writing queries for the ZK Query Language.

This was the implicit meaning already, but you're right.

Reply:
Done